.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
This is a library skeleton, so we should forbid package-locks, since they are not included in the published package, leading to a discrepancy between the tests and the fresh install result.

This way they can potentially hide upstream bugs that our CI will not reveal, even though a fresh `npm install` may not work for a consumer.